examples/base_1cc/main.tf

```diff
@@ -111,6 +111,7 @@ locals {
 CC_URL=${var.cc_vm_prov_url}
 AZURE_VAULT_URL=${var.azure_vault_url}
 HTTP_PROBE_PORT=${var.http_probe_port}
+CLIENT_ID=${module.cc_identity.managed_identity_client_id}
```

Comment:
As the new comments in the Janus's PR, we want to change the client_id in userdata.cfg to "azure_managed_identity_client_id"

Reply:
@eliassun fixed in latest commit